src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -152,4 +166,17 @@ private static boolean isSwitchExpression(DetailAST ast) {
         return ast.getParent().getType() == TokenTypes.EXPR
                 || ast.getParent().getParent().getType() == TokenTypes.EXPR;
     }
+
+    /**
+     * Checks if the case contains null label.
+     *
+     * @param ast the switch statement we are checking
+     * @return returnValue the ast of null label
+     */
+    private static boolean hasNullCaseLabel(DetailAST ast) {
+        return ast.getFirstChild() != null
+                && ast.getFirstChild().getFirstChild() != null
+                && ast.getFirstChild().getFirstChild().getType() == TokenTypes.LITERAL_NULL;
```

Comment:
```suggestion
        final DetailAst firstChild = ast.getFirstChild();
        return firstChild != null
                && TokenUtil.ifOfType(firstChild.getFirstChild(), TokenTypes.LITERAL_NULL);
```

We can use utility method to reduce redundant code here, and also avoid extra method calls by extracting firstChild.

Reply:
Done @nrmancuso